actions/checkout/action.yaml

```diff
@@ -0,0 +1,68 @@
+name: "Checkout repository"
+description: "Composite action for checking out a repository"
+inputs:
+  cache:
+    description: "Describes if the repository is using any LFS files"
+    required: false
+    default: "true"
+  fetch-depth:
+    description: "Number of commits to fetch. 0 indicates all history for all branches and tags"
+    required: false
+    default: "1"
+  lfs:
+    description: "Describes if the repository is using any LFS files"
+    required: false
+    default: "false"
+  persist-credentials:
+    description: "Whether to configure the token or SSH key with the local git config"
+    required: false
+    default: "true"
+  ref:
+    description: "The branch, tag or SHA to checkout"
+    required: false
+    default: ""
+  repository:
+    description: "The repository name with owner"
+    required: false
+    default: ${{ github.repository }}
+  token:
+    description: "Personal access token (PAT) used to fetch the repository"
+    required: false
+    default: ${{ github.token }}
+
+outputs:
+  lfs-cache-hit:
+    description: "A boolean value to indicate an exact match was found for the key"
+    value: ${{ steps.cache-lfs.outputs.cache-hit }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
```

Comment:
Please make sure to update to `v4` before merge because of [issues with v3](https://github.com/actions/checkout/issues/1448)